src/applications/proxy-rewrite/proxy-rewrite-entry.jsx

```diff
@@ -16,6 +16,16 @@ import startFeedbackWidget from '../../platform/site-wide/feedback';
 // import startAnnouncementWidget from '../../platform/site-wide/announcements';
 import startVAFooter from '../../platform/site-wide/va-footer';
 
+import redirects from './otherDomainRedirects.json';
+
+const match = redirects
+  .find(redirect => redirect.domain === window.location.host
+    && redirect.src === window.location.pathname);
+
+if (match) {
+  window.location.href = `https://www.va.gov${match.dest}`;
+}
```

Comment:
Does `preview` need to be accounted for?

I think it's pretty clear what's happening here, but maybe a comment/variable name that helps document why this redirect exists. `brandConsolidationRedirect` is pretty lengthy, but maybe something like that.